### PR TITLE
LUN-383: Fav icon displaced

### PR DESCRIPTION
**Pull request by f1sh1918**

### CODE REVIEW

File: release-notes/unreleased/LUN-383-fav-icon-displaced.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-383
+show_in_stores: false
+platforms:
+  - android
+  - ios
+de: Das Favoriten Icon in Übungen wurde korrekt positioniert.
```
Comment (SteffiStuffel) [inline release-notes/unreleased/LUN-383-fav-icon-displaced.yml:6]: (optional) I like when release notes sound more optimistic and less "buggy". E.g. "Die Positionierung des Icons wurde optimiert".

**Review by SteffiStuffel — APPROVED** [review]
Still not perfect, but fine for me.
icon

File: ios/Podfile.lock [hunk 2]
```diff
@@ -638,4 +638,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ca4eb736278d4211ffaa5cef62299ef8fc2137ca
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```
Comment (steffenkleinle) [inline ios/Podfile.lock:641]: I think this should be reverted

**Review by steffenkleinle — APPROVED** [review]

File: ios/Podfile.lock (hunk 2 above)
Comment (f1sh1918) [reply ios/Podfile.lock:641]: alright but it should be updated

**Comment by f1sh1918** [comment]
> Still not perfect, but fine for me. icon

This relies to how the picture is positioned. Works different on android. On ios it looks way better. Maybe we could create a ticket for that picture issue

File: ios/Podfile.lock (hunk 2 above)
Comment (steffenkleinle) [reply ios/Podfile.lock:641]: Should be updated together with gems :) So yes, please do it in a separate PR

**Pull request merged**